### ISSUE TITLE
Error:Annotation directory not found home/hp2/jx/darkflow-master/voc2007/train/VOCdevkit/VOC2007/Annotations .

### ISSUE THREAD

**Issue by junxuezheng** [thtrieu/darkflow#612]
hp2@hp2:~/jx/darkflow-master$ flow --model cfg/yolo-voc-new.cfg --train --dataset "~/voc2007/train/VOCdevkit/VOC2007/JPEGImages" --annotation "home/hp2/jx/darkflow-master/voc2007/train/VOCdevkit/VOC2007/Annotations"

Parsing cfg/yolo-voc-new.cfg
Loading None ...
Finished in 0.00010037422180175781s

Building net ...
Source | Train? | Layer description                | Output size
-------+--------+----------------------------------+---------------
       |        | input                            | (?, 416, 416, 3)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 416, 416, 32)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 208, 208, 32)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 208, 208, 64)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 104, 104, 64)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 104, 104, 128)
 Init  |  Yep!  | conv 1x1p0_1  +bnorm  leaky      | (?, 104, 104, 64)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 104, 104, 128)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 52, 52, 128)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 52, 52, 256)
 Init  |  Yep!  | conv 1x1p0_1  +bnorm  leaky      | (?, 52, 52, 128)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 52, 52, 256)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 26, 26, 256)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 26, 26, 512)
 Init  |  Yep!  | conv 1x1p0_1  +bnorm  leaky      | (?, 26, 26, 256)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 26, 26, 512)
 Init  |  Yep!  | conv 1x1p0_1  +bnorm  leaky      | (?, 26, 26, 256)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 26, 26, 512)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 13, 13, 512)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 13, 13, 1024)
 Init  |  Yep!  | conv 1x1p0_1  +bnorm  leaky      | (?, 13, 13, 512)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 13, 13, 1024)
 Init  |  Yep!  | conv 1x1p0_1  +bnorm  leaky      | (?, 13, 13, 512)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 13, 13, 1024)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 13, 13, 1024)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 13, 13, 1024)
 Load  |  Yep!  | concat [16]                      | (?, 26, 26, 512)
 Load  |  Yep!  | local flatten 2x2                | (?, 13, 13, 2048)
 Load  |  Yep!  | concat [26, 24]                  | (?, 13, 13, 3072)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 13, 13, 1024)
 Init  |  Yep!  | conv 1x1p0_1    linear           | (?, 13, 13, 125)
-------+--------+----------------------------------+---------------
Running entirely on CPU
cfg/yolo-voc-new.cfg loss hyper-parameters:
	H       = 13
	W       = 13
	box     = 5
	classes = 20
	scales  = [1.0, 5.0, 1.0, 1.0]
Building cfg/yolo-voc-new.cfg loss
Building cfg/yolo-voc-new.cfg train op
2018-03-05 20:26:09.728497: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-03-05 20:26:10.046177: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:892] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2018-03-05 20:26:10.046635: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1030] Found device 0 with properties: 
name: TITAN X (Pascal) major: 6 minor: 1 memoryClockRate(GHz): 1.531
pciBusID: 0000:04:00.0
totalMemory: 11.90GiB freeMemory: 8.18GiB
2018-03-05 20:26:10.111958: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:892] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2018-03-05 20:26:10.112504: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1030] Found device 1 with properties: 
name: Quadro K420 major: 3 minor: 0 memoryClockRate(GHz): 0.8755
pciBusID: 0000:03:00.0
totalMemory: 1.94GiB freeMemory: 1.59GiB
2018-03-05 20:26:10.408616: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:892] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2018-03-05 20:26:10.409067: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1030] Found device 2 with properties: 
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.582
pciBusID: 0000:84:00.0
totalMemory: 10.91GiB freeMemory: 10.60GiB
2018-03-05 20:26:10.409118: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1045] Device peer to peer matrix
2018-03-05 20:26:10.409148: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1051] DMA: 0 1 2 
2018-03-05 20:26:10.409158: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1061] 0:   Y N N 
2018-03-05 20:26:10.409164: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1061] 1:   N Y N 
2018-03-05 20:26:10.409170: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1061] 2:   N N Y 
2018-03-05 20:26:10.409180: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1120] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: TITAN X (Pascal), pci bus id: 0000:04:00.0, compute capability: 6.1)
2018-03-05 20:26:10.409188: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1107] Ignoring gpu device (device: 1, name: Quadro K420, pci bus id: 0000:03:00.0, compute capability: 3.0) with Cuda multiprocessor count: 1. The minimum required count is 8. You can adjust this requirement with the env var TF_MIN_GPU_MULTIPROCESSOR_COUNT.
2018-03-05 20:26:10.409195: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1120] Creating TensorFlow device (/device:GPU:1) -> (device: 2, name: GeForce GTX 1080 Ti, pci bus id: 0000:84:00.0, compute capability: 6.1)
Finished in 10.376970291137695s

Enter training ...
Error: Annotation directory not found home/hp2/jx/darkflow-master/voc2007/train/VOCdevkit/VOC2007/Annotations .

what is the wrong?how should i do ?there are .xml file in Annotations . thanks

**Comment by sandeeprepakula** [comment]
Are you sure the path is correct? I guess you are missing / in the beginning. I guess the path should be "/home/hp2/jx/darkflow-master/voc2007/train/VOCdevkit/VOC2007/Annotations"

**Comment by junxuezheng** [comment]
@sandeeprepakula ,thanks  
flow --model cfg/yolo-new.cfg --train --dataset "~/VOCdevkit/VOC2007/JPEGImages" --annotation "~/VOCdevkit/VOC2007/Annotations"    should be    flow --model cfg/yolo-new.cfg --train --dataset "VOCdevkit/VOC2007/JPEGImages" --annotation "VOCdevkit/VOC2007/Annotations"

**Comment by linydf** [comment]
Hi，can you tell where can i download the yolo-new.cfg or how to make yolo-new.cfg,ths

**Comment by junxuezheng** [comment]
@linydf 
yolo-new.cfg is yolo-voc.cfg. i copy and paste.

**Comment by linydf** [comment]
@junxuezheng I used above command,but met this error,are you success???
![5656](https://user-images.githubusercontent.com/32791070/38533117-76d5fa74-3caa-11e8-872d-346b9b9b7c96.jpg)
